.release/buildspec_e2e.yml

```diff
@@ -210,16 +246,24 @@ phases:
       - |
         if [ ! -z "$TESTENV_REGION" ]; then
           account=$(aws secretsmanager get-secret-value --secret-id ${TESTENV_ACCOUNT} | jq '.SecretString' -r)
-          printf "[profile e2etestenv]\nregion=$TESTENV_REGION\n" >> /tmp/.aws/config
-          printf "[e2etestenv]\n" >> /tmp/.aws/credentials
+          printf "[profile test]\nregion=$TESTENV_REGION\n" >> /tmp/.aws/config
+          printf "[test]\n" >> /tmp/.aws/credentials
           printf "aws_access_key_id=$(echo $account | jq '.access_key' -r)\n" >> /tmp/.aws/credentials
           printf "aws_secret_access_key=$(echo $account | jq '.secret_access_key' -r)\n\n" >> /tmp/.aws/credentials
         fi
       - |
         if [ ! -z "$PRODENV_REGION" ]; then
           account=$(aws secretsmanager get-secret-value --secret-id ${PRODENV_ACCOUNT} | jq '.SecretString' -r)
-          printf "[profile e2eprodenv]\nregion=$PRODENV_REGION\n" >> /tmp/.aws/config
-          printf "[e2eprodenv]\n" >> /tmp/.aws/credentials
```

Comment:
We probably need to update our contribute.md for this change. https://github.com/aws/copilot-cli/blob/mainline/CONTRIBUTING.md#building-and-testing

Reply:
good catch!